src/NzbDrone.Core/ImportLists/LidarrLists/LidarrLists.cs

```diff
@@ -29,7 +32,6 @@ public override IEnumerable<ProviderDefinition> DefaultDefinitions
                 yield return GetDefinition("Apple Music New Albums", GetSettings("apple-music/album/new"));
                 yield return GetDefinition("Billboard Top Albums", GetSettings("billboard/album/top"));
                 yield return GetDefinition("Billboard Top Artists", GetSettings("billboard/artist/top"));
-                yield return GetDefinition("Last.fm Top Albums", GetSettings("lastfm/album/top"));
```

Comment:
Should this be deleted?

Reply:
Yea, this chart doesn't work correctly, we are using a tag that only has certain subset of artists thus it gives obscure results.